tests/Type/data/relation.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
We already have `model-relations.php` so we can just move these to that file.

Reply:
Fixed.